client/components/checklist/style.scss

```diff
@@ -188,83 +184,79 @@
 	.spinner {
 		display: none;
 		position: absolute;
-		top: 12px;
+		top: 17px;
 		left: 24px;
 	}
 
-	&-warning-background {
+	.checklist__task-warning-background {
 		display: block;
 		position: absolute;
-		top: 18px;
-		left: 28px;
+		top: -1px;
+		left: 0;
 		width: 18px;
 		height: 18px;
 		border-radius: 16px;
 		background: var( --color-white );
 		cursor: pointer;
 	}
 
-	&-primary {
-		display: flex;
-		flex-direction: column;
-		flex: 1 1 90%;
+	.checklist__task-title {
+		text-align: left;
 	}
 
-	&-secondary {
-		display: flex;
-		flex: 1 1 10%;
-		align-items: flex-start;
-		padding-left: 10px;
+	.checklist__task-title-button.button {
+		position: relative;
+		display: block;
+		padding: 16px $task-right-padding 4px $task-left-padding;
+		width: 100%;
+		text-align: left;
+		color: var( --color-neutral-dark );
+		font-weight: 600;
+		font-size: 16px;
+		border-radius: 0;
 
-		@include breakpoint( '<480px' ) {
-			padding-left: 0;
-			align-items: center;
+		.checklist__toggle-affordance {
```

Comment:
I'm personally a bit confused by the term 'affordance' :sweat_smile: Is that a standard UI design term, without any good alternatives? Are there maybe some synonyms that are clearer to us developer folk? :smile:

Reply:
yes, it is a standard UI term. We could change it to `checklist__toggle-icon` — that's probably clearer.

Reply:
I think that would be clearer yeah 👍

Reply:
Updated!